commitlint.config.js

```diff
@@ -0,0 +1,135 @@
+module.exports = {
```

Comment:
Isn't this equivalent to `module.exports = {extends: ['@commitlint/config-conventional']}` ? I think we should only consider "ejecting" only if we need custom a custom config, and I see that you have already installed `@commitlint/config-conventional` in `package.json` so we might as well use it.